src/Silk.Data/MediatR/Greetings/GetGuildGreetings.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Mapster;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Remora.Rest.Core;
+using Silk.Data.DTOs.Guilds.Config;
+
+namespace Silk.Data.MediatR.Greetings;
+
+public static class GetGuildGreetings
+{
+    public record Request(Snowflake GuildID) : IRequest<List<GuildGreeting>>;
+
+    internal class Handler : IRequestHandler<Request, List<GuildGreeting>>
+    {
+        private readonly IDbContextFactory<GuildContext> _dbContextFactory;
```

Comment:
Is there a reason of using DBCF over `GuildContext`?

Reply:
Using the DBCF and creating a new instance per handler call prevents DbContext threading issues. Mostly on Blazor's behalf when needing to request more than 1 thing at a time through MediatR